app/models/source.rb

```diff
@@ -4,10 +4,11 @@ class Source < ActiveRecord::Base
   has_many :project_sources
   has_many :projects , through: :project_sources
   belongs_to :user
```

Comment:
Sawy, pay attention that this `belongs_to` here will use the `user_id` in the source - so, `source.user` will return the creator, not the related user, which I think is the case, but just to clarify.